integrations/terraform/envs/portal/config/data_glossary/data_outputs.yaml

```diff
@@ -1,17 +1,20 @@
 raw_test:
-  s3:
-    - bucket_name: datalake
-      path: /raw/test
-  owner: ["test-project-1"]
+  S3:
```

Comment:
Why did you not keep this as a list, we need to be flexible for the future. Can be that we need to adapt the UX in the future